Automattic/jetpack
# Forms: Update dashboard body font-size

## Proposed changes:

* Updates the Forms dashboard body font-size to `14px` and reduces the response list row height to a more compact version

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Check if the changes described above were applied correctly
